examples/user_guide/16-Streaming_Data.ipynb

```diff
@@ -206,357 +202,6 @@
     "Note that when using the ``Buffer`` stream the view will always follow the current range of the data by default, by setting ``buffer.following=False`` or passing following as an argument to the constructor this behavior may be disabled."
    ]
   },
-  {
```

Comment:
There are a couple of nice use cases in this whole Streamz part that is being removed, have you considered porting them to Pipe/Buffer?

Reply:
Yes, but it isn't worth the effort, in my opinion. You are welcome to give it a go.